packages/core/README.md

```diff
@@ -30,6 +30,19 @@ import '@react95/core/GlobalStyle';
 import '@react95/core/themes/win95.css';
 ```
 
+If it doesn't work, complete the import like this:
+
+```js
+import * as GlobalStyle from '@react95/core/GlobalStyle';
+import * as themes from '@react95/core/themes/win95.css';
+```
+
+And use them at least once somewhere. The easiest way to do this is by:
+
+```js
+console.log({ GlobalStyle }, { themes });
+```
+
```

Comment:
This has become obsolete since #449. 
We can remove this section 🧹

Reply:
> This has become obsolete since #449. We can remove this section 🧹

This provides a clear and concise explanation and solution for the problems we have encountered in recent days!
Thank you for pointing out the cause of the problem!